packages/simplebar/src/index.ts

```diff
@@ -172,18 +172,16 @@ export default class SimpleBar extends SimpleBarCore {
           if (
             (removedNode as Element).getAttribute('data-simplebar') === 'init'
           ) {
-            SimpleBar.instances.has(removedNode) &&
-              !document.documentElement.contains(removedNode) &&
-              SimpleBar.instances.get(removedNode).unMount();
+            !document.documentElement.contains(removedNode) &&
+              SimpleBar.instances.get(removedNode)?.unMount();
           } else {
             Array.prototype.forEach.call(
               (removedNode as Element).querySelectorAll(
                 '[data-simplebar="init"]'
               ),
               (el) => {
-                SimpleBar.instances.has(el) &&
-                  !document.documentElement.contains(el) &&
-                  SimpleBar.instances.get(el).unMount();
+                !document.documentElement.contains(el) &&
+                  SimpleBar.instances.get(el)?.unMount();
```

Comment:
I suppose since we have `SimpleBar.instances.get(el)?` we don't need the `SimpleBar.instances.has(el)`. Is that it or there is more to it?

Reply:
That’s it, and we now need the `?` because TypeScript now understands that `.get` returns `SimpleBar | undefined` instead of `any`. (`any` effectively means “unsoundly disable type checking here”, vs. `unknown` which is “soundly require additional narrowing before allowing more specific use of this value”. You never want `any`.)